draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -5032,6 +5082,11 @@ Content-Range: exampleunit 11.2-14.3/25
    through one or more proxies, which can then be secured using TLS
    (Transport Layer Security, <xref target="RFC8446"/>).
 </t>
+<t>
+   Because CONNECT changes the request/response nature of an HTTP connection,
+   specific HTTP versions might have different ways of mapping its semantics
+   into the protocol's wire format.
+</t>
```

Comment:
I think you missed my second comment that this paragraph doesn't say anything useful. AIUI, CONNECT was recently extended to allow an origin server to process it, which wasn't about the wire format.

I would prefer that we update the first paragraph to be less specific to 1.1 and then reference the CONNECT extension as an example of an allowed semantic extension because the method had no applicability for an origin server before.

I am also fine with merging this now and fixing CONNECT later.

Reply:
RFC 7231 already permits origin servers to process CONNECT.

> An origin server that receives a CONNECT request for itself MAY respond with a 2xx (Successful) status code to indicate that a connection is established.  However, most origin servers do not implement CONNECT.

But yes, that extension stretches the bounds of this description.